shared/src/profileData.ts

```diff
@@ -37,6 +37,7 @@ export interface IndustrySpecificData {
   readonly residentialConstructionType: ResidentialConstructionType;
   readonly employmentPersonnelServiceType: EmploymentAndPersonnelServicesType;
   readonly employmentPlacementType: EmploymentPlacementType;
```

Comment:
For testing only: if you want to validate the test. Will be removed before a merge